packages/terra-form-input/tests/jest/Input.test.jsx

```diff
@@ -17,6 +17,14 @@ describe('Input', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should render placeholder when showMpageFusionPlaceholder is passed into the Input', () => {
+    const input = <Input defaultValue="foo" id="placeholderText" ariaLabel="label" placeholder="Enter The Text" showMpageFusionPlaceholder />;
+    const wrapper = shallow(input);
+    const placeholderInput = wrapper.find('input#placeholderText');
+    expect(placeholderInput.props()).toHaveProperty('placeholder', 'Enter The Text');
```

Comment:
we need to test the other scenario as well here ensure  placeholder is set to null when you pass placeholder without showMpageFusionPlaceholder prop